App/types/navigators.ts

```diff
@@ -72,6 +72,7 @@ export type CredentialStackParams = {
 export type HomeStackParams = {
   [Screens.Home]: undefined
   [Screens.Notifications]: undefined
+  [Screens.ProofRequestAttributeDetails]: { proofId: string; attributeName: string }
```

Comment:
Maybe I'm missing something but this seems to be duplicated with line 90 below?

Reply:
Each stack have their own type, but they are very similar. There is probably an opportunity to refactor the code. But if it is alright, I will leave that for another work.